IntegrationTests/Services/AWSBedrockRuntimeIntegrationTests/BedrockAPIKeyTests.swift

```diff
@@ -14,7 +14,7 @@ final class BedrockAPIKeyIntegrationTests: XCTestCase {
     let envVarName = "AWS_BEARER_TOKEN_BEDROCK"
     let apiKeyDuration: TimeInterval = 600.0
 
-    func xtest_apiKey_createsAPIKeyAndCallsWithIt() async throws {
+    func test_apiKey_createsAPIKeyAndCallsWithIt() async throws {
```

Comment:
Re-enabling this integration test; it's the one which was causing problems on systems that assume a role before testing.

Reply:
BTW I re-ran this test on the internal build system, it now passes after the refactor.